analyzer/src/funTest/assets/projects/synthetic/spdx/subproject-conan/subproject/subproject-with-conan-reference.spdx.yml

```diff
@@ -0,0 +1,21 @@
+SPDXID: "SPDXRef-DOCUMENT"
+spdxVersion: "SPDX-2.2"
+creationInfo:
+  created: "2022-07-06T12:00:00Z"
+  creators:
+    - "Organization: Example Inc."
+  licenseListVersion: "3.9"
+name: "sub-conan-0.1.0"
+dataLicense: "CC0-1.0"
+documentNamespace: "http://spdx.org/spdxdocs/spdx-document-xyz"
+documentDescribes:
+  - "SPDXRef-Package-subproject-conan"
+packages:
+  - SPDXID: "SPDXRef-Package-subproject-conan"
+    copyrightText: "Copyright (c) 2020 Example Inc."
+    downloadLocation: "git+ssh://gitlab.example.com:3389/products/conan-project.git@b2c358080011af6a366d2512a25a379fbe7b1f78"
+    licenseConcluded: "NOASSERTION"
+    licenseDeclared: "NOASSERTION"
+    name: "ConanProject"
+    versionInfo: "1.0.0"
+    packageFileName: "./conanfile.py?scope=requires"
```

Comment:
I fear this "made-up" syntax renders this file incompatible with any other SPDX parsers. What's the reason for trying it like this rather than relying on an `externalRefs` entry with a "purl" for this package, which *can* easily contain a query string / qualifier as part of the URI?

Reply:
I like the idea. So, `packageFileName` would be a plain file name, and there would be an additional `externalRef` of type `Purl`, from which only the `scope` query parameter would be evaluated, right?

Reply:
Yes, exactly, that's what I'd propose (and how I believe it was discussed Bosch-internally before; that's why I was asking whether there's maybe a reason to deviate from that proposal, but probably it was just forgotten / misunderstood).

Reply:
FYI, here's an example (by @jsteinhofff from Bosch) how PURLs for COnan should look like: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#conan

Reply:
I have changed the implementation accordingly in a new commit.